presto-main/etc/catalog/hive.properties

```diff
@@ -6,4 +6,6 @@
 #
 
 connector.name=hive-hadoop2
-hive.metastore.uri=thrift://localhost:9083
+#hive.metastore.uri=thrift://localhost:9083
+hive.metastore.discovery.type=dynamic
+hive.metastore.discovery.uri={"url":"http://localhost:5436/discover","headers":{"Rpc-Service":"hms-general_staging_neon","Rpc-Caller":"hs2-general_staging_neon"}}
```

Comment:
Is this change necessary?

Reply:
Added the configs related to dynamic discovery in comments and keeping the static one as the default implementation